config/locales/en.yml

```diff
@@ -2586,6 +2586,20 @@ en:
         edit:
           helper_description: Please continue to see what else we can guide you to qualify for.
           title: Looks like you’re not eligible for this credit.
+      nj_medical_expenses:
+        edit:
+          help_text: If you do not want to claim this deduction, click “Continue”.
+          includes_whom: This includes expenses for yourself, your spouse, and your dependents.
+          label: Non-reimbursed medical expenses paid in %{filing_year}
+          learn_more_content_html: "<li>Physicians, dental, and other medical fees</li><li>Prescription eyeglasses and contact lenses</li><li>Hospital care</li><li>Nursing care</li><li>Medicines and drugs</li><li>Prosthetic devices</li><li>X‑rays and other diagnostic services conducted by or directed by a physician or dentist</li><li>Amounts paid for transportation primarily for and essential to medical care</li><li>Insurance (including amounts paid as premiums under Part B of Title XVIII of the Social Security Act, relating to supplementary medical insurance for the aged) covering medical care</li>"
```

Comment:
[dust/pebble] Would be nice to format this using the `|` so that each `li` can be its only line in the yaml